### PR TITLE
loki-21: Added Search to the admin queue page.

### PR DESCRIPTION
**Pull request by rfxu**
loki-13:  Added a message showing x of x entries to indicate no search results.

### CODE REVIEW

File: frontend/src/app/admin-dashboard/admin-dashboard.component.html [hunk 1]
```diff
@@ -37,17 +55,22 @@ <h3>
...
 				<td>{{customer.firstName}}</td>
 				<td>{{customer.lastName}}</td>
 				<td>{{getStatusText(customer.status)}}</td>
 				<td>{{customer.uniqueCode}}</td>
 				<td><a routerLink="/admin/detail/{{customer.id}}"> View Detail</a></td>
 			</tr>
 			</tbody>
+			<tfoot>
+				<tr>
+					<td colspan="5"><div class="color-gray">Showing <b>{{filteredCustomerStatuses.length}}</b> of <b>{{customerStatuses.length}}</b> entrices</div>
```
Comment (jessetaylornci) [inline frontend/src/app/admin-dashboard/admin-dashboard.component.html]: "entries" is mispelled

File: frontend/src/app/admin.service.spec.ts [hunk 2]
```diff
@@ -57,40 +63,49 @@ describe('AdminService', () => {
...
+		let request = httpMock.expectOne('/rejectUpload/111222');
+		request.flush("Test");
 			
-	 httpMock.verify();
-  });
+		httpMock.verify();
+	});
+  
+  
+    it('should call updateUploadCommen', (done) => {
```
Comment (jessetaylornci) [inline frontend/src/app/admin.service.spec.ts]: updateUploadCommen =-> updateUploadCommen**t**

File: frontend/src/app/admin-dashboard/admin-dashboard.component.html (hunk 1 above)
Comment (rfxu) [reply frontend/src/app/admin-dashboard/admin-dashboard.component.html]: Fixed

File: frontend/src/app/admin.service.spec.ts (hunk 2 above)
Comment (rfxu) [reply frontend/src/app/admin.service.spec.ts]: Fixed

**Pull request merged**